contents/handbook/people/team-structure/infrastructure.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Team Infrastructure
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## People
+
+- James Greenhill (Team lead, Data/Infra Engineer)
+- Guido Iaquinti (Site Reliability Engineer)
+- Harry Waye (Full Stack Engineer)
+
+## Mission Statement
+Make deploying, scaling, and managing PostHog easy, fast, and reliable.
+
+## Q3 2022 Goals
+* **Proposed Objective**:
+    * Lots of customers are self-serving PostHog Cloud in the EU
```

Comment:
I would explicitly add here a point to include all the work currently needed to _"keep the lights on"_. Here’s a non conclusive list of work that I already foresee for the next quarter:

* PostHog Cloud: general service maintenance of the service 

* PostgreSQL: 
    * as seen in the past, we often experience performance degradation because of new application behaviors/functionalities. Our team has been historically the first responder to troubleshoot, evaluate and often also fix those regressions.  

    * the overall dataset (and specifically the `posthog_person` table) is massively growing week after week. We don’t have a long-term strategy on how to mitigate this issue that might become more urgent as time passes.

* Clickhouse: like for PostgreSQL, we are often the first responder to troubleshoot, evaluate, and often also fix performance regressions to this tier.

* Helm chart: in the last couple of months we’ve been focusing on rebuilding our PostHog Cloud footprint & move our stateless workloads to EKS. This made the development and progress of our Helm chart stalling and we’ll need to pay out some technical debt. Example:  Kubernetes v1.25 will be released in August and we already have 100% confidence the chart will not work for that version.

....